packages/tsx-editor/src/components/Editor.tsx

```diff
@@ -1,12 +1,19 @@
 import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
-import React from 'react';
+import * as React from 'react';
 import { IEditorProps } from './Editor.types';
 
-export class Editor extends React.Component<IEditorProps> {
-  private editor: monaco.editor.IStandaloneCodeEditor | undefined;
-  private editorRef = React.createRef<HTMLDivElement>();
+export const Editor: React.FunctionComponent<IEditorProps> = (props: IEditorProps) => {
+  const ref = React.useRef<HTMLDivElement>(null);
+  const { width, height, onChange, code, language } = props;
+  const style = { width, height };
```

Comment:
>style [](start = 8, length = 5)

Why is this called "style" instead of "size"? Are we expecting to have more properties here?

Reply:
Could go either way on this. Just opted for consistency in naming since divs accept a style attribute as a prop and I pass my style in there (see the returned div)).

Reply:
I think calling it `style` makes sense to match the prop name. Or you could just set it inline instead of putting it in a variable first, since it's only a couple properties.